### PR TITLE
Added Star Equals symbol

### PR DESCRIPTION
**Pull request by Xraycat**
Added Star Equals symbol

### CODE REVIEW

**Comment by Xraycat** [comment]
Added Star Equals symbol

**Review by DenverCoder1 — COMMENTED** [review]
Spaces need to be replaced with tabs

File: dictionary.txt [hunk 1]
```diff
@@ -930,3 +930,4 @@ _rp	₎
 \daleth	ℸ	
 ^epsilon	ᵋ	
 \rightangle	∟	
+\starequals	≛
```
Comment (DenverCoder1) [inline dictionary.txt:933]: Suggestion: remove the `s`. I was able to find references to a LaTeX shortcut called `\starequal` but not `\starequals`

http://www.sput.nl/software/latex-symbols.html

```suggestion
\starequal	≛	
```

**Review by DenverCoder1 — APPROVED** [review]
Great! :+1:

**Pull request merged**